resources/views/inputs/checkbox.blade.php

```diff
@@ -9,10 +9,11 @@
     'value'                 => null,
     'checked'               => false,
     'disabled'              => false,
+    'right'                 => false
 ])
 
 <div class="{{ $class }}">
-    <div class="flex relative {{ $checkboxPositionClass }}">
+    <div class="flex relative {{ $checkboxPositionClass }} @if($right) flex-row-reverse @endif">
```

Comment:
I don't know if we want to change `checkboxPositionClass` with something like `verticalPosition="top|middle|bottom"` @ItsANameToo @Highjhacker what's your thought?

Reply:
@leMaur I think it's good to have that too to limit the amount of options. Will require a change in MSQ though as the property is currently being used there, but that will be a quick change